### ISSUE TITLE
Confusing name conference_id in versions table

### ISSUE THREAD

**Issue by namangupta01** [openSUSE/osem#1728]
In file https://github.com/openSUSE/osem/blob/master/app/models/role.rb#L6 when name of the role is **organization_admin** then we have resource_type equals to **organization** and **resource_id** contains contains the id of the **organization** but in this line https://github.com/openSUSE/osem/blob/master/app/models/role.rb#L6
**conference_id** is given a value of **resource_id**, where resource_id contains the value of organization id in this case which is confusing as conference_id now contains the id of the organization when name of role is organization_admin.

**Comment by differentreality** [comment]
@shlok007 did you, by any chance, already run into this?

**Comment by shlok007** [comment]
Yes, this is definitely something I ran into. However, I am not sure if we should label it as a bug because it does not break anything at the moment, right? This `conference_id` name however might confuse someone else reading our code.

**Comment by differentreality** [comment]
Well if we have the attribute conference_id and it ends up holding the organization_id, I'd say this is a bug. The fact that things have not broken completely is completely coincidental.

**Comment by shlok007** [comment]
I remember writing code keeping this in mind that for `organisation admin` role, `conference_id` column will contain organisation_id instead of conference_id : https://github.com/openSUSE/osem/blob/master/app/views/admin/versions/_object_desc_and_link.html.haml#L144 . ( I also did add a comment to make sure we remember to rename this later on )

**Comment by differentreality** [comment]
@shlok007 where does this happen? That org_id is assigned in conference_id attribute of roles table.
